### DISCUSSION TITLE
Metrics Insights not updating at all

### DISCUSSION THREAD

**Discussion by lulunac27a** [lowlighter/metrics#1667] [General]
When I visit Metrics Insights like [this user](https://metrics.lecoq.io/insights?user=lowlighter), it doesn't update the profile statistics since last week because the requests to insights query returns `204 No Content` instead of `200 OK` and the warning messages show `base: no data yet, retrying in 5 seconds` and other plugins with the similar messages.

**Comment by lulunac27a** [comment]
A week has passed and still not updating the profile stats at all.

**Comment by lulunac27a** [comment]
Still not working, the warning message says `base: no data yet, retrying in 5 seconds`

**Reply by lulunac27a** [reply]
Still gets the same problem for months now.

**Comment by Vault108** [comment]
@lowlighter  The website is also not working.

**Comment by Krasipeace** [comment]
I'm getting this error past few days, is this related due to the metrics website not working?
```
 ERROR: failed to solve: process "/bin/sh -c chmod +x /metrics/source/app/action/index.mjs   
&& apt-get update   && apt-get install -y wget gnupg ca-certificates libgconf-2-4   
&& wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -   
&& sh -c 'echo \"deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main\" >> /etc/apt/sources.list.d/google.list'   
&& apt-get update   
&& apt-get install -y google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst fonts-freefont-ttf libxss1 libx11-xcb1 libxtst6 lsb-release --no-install-recommends   
&& apt-get install -y curl unzip   && curl -fsSL https://deno.land/x/install/install.sh | DENO_INSTALL=/usr/local sh   
&& apt-get install -y ruby-full git g++ cmake pkg-config libssl-dev   && gem install licensed   
&& apt-get install -y python3   && rm -rf /var/lib/apt/lists/*   && npm ci   
&& npm run build" did not complete successfully: exit code: 1
  Error: Process completed with exit code 1.
```

**Reply by lulunac27a** [reply]
Maybe the metrics website is not working, but it works when I use GitHub Actions.

**Reply by Krasipeace** [reply]
ok, that's weird i'm also using actions... ill check your config :P

**Reply by Krasipeace** [reply]
Actually, just chaging the repo from my fork to lowlighter/metrics@latest fixed the issue, and now is generating the svg.

**Reply by lulunac27a** [reply]
The metrics Insights website is still not fetching my data for more than a month now

**Reply by e-kotov** [reply]
i sorted this out in my repo ( https://github.com/e-kotov/metrics ), will do a PR now

**Reply by e-kotov** [reply]
PR here https://github.com/lowlighter/metrics/pull/1682

**Reply by e-kotov** [reply]
Oh, there was also another PR fixing the same issue: https://github.com/lowlighter/metrics/pull/1677

**Reply by lulunac27a** [reply]
These pull requests are not from the metrics insights website, but uses GitHub Actions to update the metrics.

**Comment by lulunac27a** [comment]
Now it encountered the same problem again.
